Prac 5/word_occurences.py

```diff
@@ -0,0 +1,10 @@
+user_string = input("Text:").split()
+strings_dict = {}
+
+# for string in user_string:
+#     if string in strings_dict:
+#         strings_dict[string] += 1
+#     else:
+#         strings_dict[string] = 1
+#     string_frequency = strings_dict[string]
+#     print("{} : {}".format(string, string_frequency))
```

Comment:
Is _string_ valid outside of the for loop on line 4? Perhaps a **_list_** is **_key_** to printing all the words in the dictionary?